homework/english/fyodorov/lecture2/task1_grammar.md

```diff
@@ -0,0 +1,32 @@
+# Упражнения Past Simple и Present Perfect
+### Упражнение 1. Put the verbs in brackets in the Past Simple or in the Present Perfect.
+1. I have just had a nice pot of coffee. Would you like a cup?
+2. I haven't seen Steve this morning yet.
+3. Carol and I are old friends. I have known her since I was a freshman in high school.
+4. Maria have had a lot of problems since she came to this country.
+5. I went to Paris in 2003 and 2006.
+6. A car came round the corner and I have jumped out of the way.
+7. Don’t throw the paper away because I haven't read it yet.
+8. Is Jim going to eat lunch with us today? — No. He has already eaten. He ate lunch an hour ago.
+9. Since we started doing this exercise, we have completed some sentences.
+10. I have never been to Italy.
+### Упражнение 2. Put the verbs in brackets in the Past Simple or in the Present Perfect.
+1. Have You ever worked in a shop?
+2. I  have worked at my uncle’s shop when I was younger.
+3. It’s the first time I have been on a ship.
+4. Ann is looking for her key. She can’t find it. She has lost her key.
+5. How many symphonies has Beethoven composed?
+6. Look! Somebody has spilt ink on the notebook.
+7. Have You had a holiday this year yet?
+8. Have You seen any good films recently?
+9. He hasn't had any problems since he came here.
+### Упражнение 3. Раскройте скобки, употребляя глаголы в Past Simple или Present Perfect
+1. I cut some flowers from my garden yesterday. I have cut lots of flowers from my garden so far this summer.
+2. I haven't seen Tom lately.
+3. The artist drew a picture of sunset yesterday. She drew many pictures of sunsets in her lifetime.
```

Comment:
how Beethoven differs from an artist?

Reply:
Realy I don't understand, is lifetime Present Perfect?